tools/system_libs.py

```diff
@@ -340,13 +340,13 @@ def get_ext(self):
     """
     Return the appropriate file extension for this library.
     """
-    return 'a' if shared.Settings.WASM_BACKEND and shared.Settings.WASM_OBJECT_FILES else 'bc'
+    return '.a' if shared.Settings.WASM_BACKEND and shared.Settings.WASM_OBJECT_FILES else '.bc'
```

Comment:
Is there a particular reason for changing this?

Reply:
No totally relevant to this change.  Mostly because we include the period in other places where we use file extensions.   So its consistent with emcc.py for example, but also with os.path.splitext().